README.md

```diff
@@ -369,6 +375,40 @@ For example:
 The [Laravel Transmorpher Client](https://github.com/cybex-gmbh/laravel-transmorpher-client) will receive this information and store it. It can also create URLs with
 transformations.
 
+## PDF handling
+
+Requesting a PDF file will return the document.
+Metadata can be removed optionally by setting the `.env` key: 
+
+```dotenv
+TRANSMORPHER_DOCUMENT_REMOVE_METADATA=true
+```
+
+### Images
+
+When an image format transformation is specified, an image of a page will be returned.
+
+All available image transformations can also be applied to PDF image derivatives.
+Requesting a PDF also follows the same URL structure as images, just replace `images` with `documents`.
+
+The page which should be used as image can be specified with the `p` transformation.
+Otherwise, the first page will be used.
```

Comment:
```suggestion
By using the `p` transformation, you can specify the page to be exported.
If no according transformation has been given, the first page will be used instead.

All available image transformations can also be applied to PDF image derivatives.
Requesting a PDF also follows the same URL structure as images, just replace `images` with `documents`.
```

unnecessary bouncing between topics. would switch the lower 2 paragraphs
line 389: create images of an page
line 391: apply transformations
line 394: specify page

if you want to keep your text, it probably should be "the page `that` should be used as `an` image"....

Reply:
Changed it to your version for better flow.